dir.props

```diff
@@ -45,6 +46,8 @@
     <BuildToolsTaskDir Condition="'$(BuildToolsTargets45)' == 'true'">$(ToolsDir)net45/</BuildToolsTaskDir>
     <UseRoslynCompilers Condition="'$(UseRoslynCompilers)'=='' and '$(OSEnvironment)'!='Windows_NT'">false</UseRoslynCompilers>
   </PropertyGroup>
+
+  <Import Project="$(ToolRuntimePath)BuildVersion.targets" />
```

Comment:
From a naming convention we generally try to only import other props files in props files and targets files in other targets files. Is there any reason BuildVersion needs to be a targets file? From a quick look it doesn't actually define any targets.